ppoprf/src/ppoprf.rs

```diff
@@ -274,23 +299,22 @@ impl Client {
     false
   }
 
-  pub fn unblind(p: &Point, r_ser: [u8; 32]) -> Point {
-    let r = Scalar::from_bytes_mod_order(r_ser);
+  pub fn unblind(p: &Point, r: &CurveScalar) -> Point {
     let point = p.decompress().unwrap();
-    let r_inv = r.invert();
+    let r_inv = RistrettoScalar::from_bytes_mod_order(r.0).invert();
```

Comment:
I haven't counted the uses, but would it be more efficient for `CurveScaler` to wrap `RistrettoScalar` instead of `[u8; 32]` so the mod operation isn't necessary when converting between the two?